txbits/conf/evolutions/default/2.sql

```diff
@@ -258,12 +258,21 @@ begin
     update balances set hold = hold - new_amount
     where currency = ask.base and user_id = ask.user_id;;
 
-    update balances set hold = hold - new_amount * new_price
-    where currency = bid.counter and user_id = bid.user_id;;
+    if bid.price >  new_price then
```

Comment:
The unit test for this is here: https://github.com/txbits/txbits/pull/110

@vikstrous I see there's changes to the evolutions needed on Postgres 9.5 so I'm fine with fixing this too.